casadm/statistics_model.c

```diff
@@ -776,6 +776,17 @@ int cache_status(unsigned int cache_id, unsigned int core_id, int io_class_id,
 		goto cleanup;
 	}
 
+	if ((cache_info.info.state & (1 << ocf_cache_state_standby)) &&
```

Comment:
Again, perhaps a function/macro similar to "ocf_cache_is_standby" would be better here?